test/mocha/cli.js

```diff
@@ -55,7 +55,7 @@ describe("bin/uglifyjs", function () {
        exec(command, function (err, stdout) {
            if (err) throw err;
 
-           assert.strictEqual(stdout, "var bar=function(){function foo(bar){return bar}return foo}();\n" + 
+           assert.strictEqual(stdout, "var bar=function(){function foo(bar){return bar}return foo}();\n" +
```

Comment:
Another PR in the queue changed this same line. Could you leave the trailing space in this one just to avoid the hassle of merging later?